types/context.go

```diff
@@ -225,3 +225,22 @@ func (c Context) CacheContext() (cc Context, writeCache func()) {
 	cc = c.WithMultiStore(cms).WithEventManager(NewEventManager())
 	return cc, cms.Write
 }
+
+type sdkContextKeyType string
+
+const sdkContextKey sdkContextKeyType = "sdk-context"
+
+// WrapSDKContext attaches a Context to that Context's context.Context member
+// and returns that context. It is useful for passing a Context through methods
+// that take a generic context.Context parameter such as generated gRPC
+// methods
+func WrapSDKContext(ctx Context) context.Context {
```

Comment:
I get what this method is doing and why we need it, but the godoc is hard to grok. WDYT of the following:

```go
// WrapSDKContext returns a stdlib context.Context with the provided sdk.Context's internal
// context as a value. It is useful for passing an sdk.Context  through methods that take a
// stdlib context.Context parameter such as generated gRPC methods. To get the original
// sdk.Context back, call UnwrapSDKContext.
```

Reply:
Sure